app/controllers/purchases_controller.rb

```diff
@@ -0,0 +1,69 @@
+class PurchasesController < ApplicationController
+  before_action :set_purchase, only: %i[show edit update destroy]
+
+  # GET /purchases or /purchases.json
+  def index
+    @purchases = Purchase.all
+  end
```

Comment:
- [ ] It will be great to order your transaction by the most recent. This is a requirement for this project to be approved.  **Hint**: try chaining this to your query `order("created_at DESC")`

Reply:
- [ ] This issue has been resolved :heavy_check_mark: